backend/codesirius/problems/models/execution_constraint.py

```
@@ -0,0 +1,80 @@
+    )
+
+    class Meta:
+        verbose_name = "Execution Constraints"
+        verbose_name_plural = "Execution Constraints"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["problem", "language"], name="unique_problem_language"
+            )
+        ]
+
+    def clean(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean`

Add return type annotation: `None`

(ANN201)